src/stdune/pp.ml

```diff
@@ -186,3 +199,8 @@ let text s = Text s
 let textf fmt = Printf.ksprintf text fmt
 
 let tag t ~tag = Tag (tag, t)
+
+let enumerate l ~f =
+  vbox (concat ~sep:cut (List.map l ~f:(fun x ->
+    box ~indent:2
+      (seq (char '-') (seq space (f x))))))
```

Comment:
What does this do if `f x` is multiline?
I guess you'd want it do do this:

```
- first line
  next line
- second item
  second line of second item
```

but it doesn't look like the code would do that. (not sure)

Reply:
That's what the `box ~indent:2` is for :)

I added a test for it (and found a bug)

Reply:
Ah, sorry, I didn't understand what `indent` was doing. Looks good.